pkg/util/k8sutil/k8sutil.go

```diff
@@ -156,6 +159,34 @@ func NewFromConfig(cfg *rest.Config) (KubernetesClient, error) {
 	return kubeClient, nil
 }
 
+// SetStatus of Postgres cluster
+// TODO: eventually switch to updateStatus() for kubernetes 1.11 and above
```

Comment:
Whats with this comment?

Reply:
traveled with the code. I'll remove it. `UpdateStatus` requires a Postgresql struct with updated status. Seems easier as of now to just patch the CRD.

Reply:
I wonder if SetStatus should be called differently now that it lives inside k8sutil. Maybe `SetPgStatus` or `SetCRDStatus`